<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSurface</class>
 <widget class="QDialog" name="CreateSurface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Surface</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="SurfaceNameL">
       <property name="text">
        <string>Surface Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SurfaceNameLE">
       <property name="text">
        <string>Surface</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QLineEdit" name="editBoundary"/>
        </item>
        <item row="0" column="0" rowspan="2" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Boundary</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="buttonBoundary">
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SelectPointGroupL">
       <property name="text">
        <string>Select Point Groups</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nube de puntos</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editCloud"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="buttonAdd">
          <property name="text">
           <string>sel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="MaxlengthL">
         <property name="text">
          <string>Maximum Triangle length:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MaxlengthLE">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="OtherValueL">
         <property name="text">
          <string>Maximum Triangle Angle:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MaxAngleLE">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>90</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
